src/games/games.module.ts

```
@@ -1,4 +1,5 @@
-import { Module } from '@nestjs/common';
+import { Module, forwardRef } from '@nestjs/common';
```

ESLint

[error] Replace `Module,·forwardRef·}·from·'@nestjs/common';` with `forwardRef,·Module·}·from·'@nestjs/common';⏎`

(prettier/prettier)